### PR TITLE
BITAU-66 Use fixed keystore for signing debug build

### PR DESCRIPTION
**Pull request by ahaisting-livefront**
## 🎟️ Tracking

https://livefront.atlassian.net/browse/BITAU-66

## 📔 Objective

The goal of this PR is to use a fixed signing key for debug builds. 

## 📸 Screenshots

N/A

## ⏰ Reminders before review

- Contributor guidelines followed
- All formatters and local linters executed and passed
- Written new unit and / or integration tests where applicable
- Protected functional changes with optionality (feature flags)
- Used internationalization (i18n) for all UI strings
- CI builds passed
- Communicated to DevOps any deployment requirements
- Updated any necessary documentation (Confluence, contributing docs) or informed the documentation team

## 🦮 Reviewer guidelines

- 👍 (`:+1:`) or similar for great changes
- 📝 (`:memo:`) or ℹ️ (`:information_source:`) for notes or general info
- ❓ (`:question:`) for questions
- 🤔 (`:thinking:`) or 💭 (`:thought_balloon:`) for more open inquiry that's not quite a confirmed issue and could potentially benefit from discussion
- 🎨 (`:art:`) for suggestions / improvements
- ❌ (`:x:`) or ⚠️ (`:warning:`) for more significant problems or concerns needing attention
- 🌱 (`:seedling:`) or ♻️ (`:recycle:`) for future improvements or indications of technical debt
- ⛏ (`:pick:`) for minor or nitpick changes

### CODE REVIEW

File: app/build.gradle.kts [hunk 1]
```diff
@@ -43,7 +43,23 @@ android {
         generateLocaleConfig = true
     }
 
+    signingConfigs {
+        getByName("debug") {
+            keyAlias = "androiddebugkey"
+            keyPassword = "android"
+            storeFile = file("../keystores/debug.keystore")
```
Comment (ahaisting-livefront) [inline app/build.gradle.kts:50]: I generated a new keystore for this. We had talked a bit about using the same debug keystore as the main bitwarden app, but ultimately I'd prefer to keep them separate and protect access to the BridgeService with `knownSigner`. 

What I like about that is that the setup for `debug` and `release` will be the same: the `BridgeService` is protected behind `knownSigner`.

**Review by SaintPatrck — APPROVED** [review]

File: app/build.gradle.kts (hunk 1 above)
Comment (brian-livefront) [reply app/build.gradle.kts:50]: Yeah I think that's a good plan.

File: app/build.gradle.kts [hunk 2]
```diff
@@ -43,7 +43,23 @@ android {
...
+            storePassword = "android"
+        }
+    }
+
     buildTypes {
+
```
Comment (brian-livefront) [inline app/build.gradle.kts:56]: To match our other gradle files we can omit the extra newline here.

Comment (ahaisting-livefront) [reply app/build.gradle.kts]: Cool 👍 

b8b85b7657b03bfd049131e81b6a87291098ecc8

**Review by SaintPatrck — APPROVED** [review]

**Review by brian-livefront — APPROVED** [review]

**Pull request merged**